src/more-infos/more-info-content.html

```diff
@@ -35,11 +35,14 @@
   },
 
   stateObjChanged: function (stateObj) {
-    if (!stateObj) return;
-
-    window.hassUtil.dynamicContentUpdater(
-      this, 'MORE-INFO-' + window.hassUtil.stateMoreInfoType(stateObj).toUpperCase(),
-      { hass: this.hass, stateObj: stateObj });
+    if (!stateObj) {
+      window.hassUtil.dynamicContentUpdater(
+        this, null, { isVisible: false });
```

Comment:
This actually had cost me money! I wasn't aware that the camera stream kept running when the dialog was closed, and it kept on streaming on my mobile phone.